src/pallavi/Numbers.java

```diff
@@ -0,0 +1,60 @@
+package pallavi;
+
+public class Numbers {
+
+	public static void main(String[] args) {
+		Methods method = new Methods();
+		method.evenNumber(10, 15);
+		method.divisibleBy5(10, 30);
+		method.divisibleByFiveAndThree(5, 18);
+		method.divisibleBy7And13(5,40);
+
+
+	}
+
+	void evenNumber(int start, int end) {
+		System.out.println("Even nubers are : ");
+		int i = start;
```

Comment:
you don’t require to create extra variable, you can handle this without **i**, think on this

please refer below code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/219/files